src/test/java/uk/gov/companieshouse/ocr/api/image/extracttext/TextConversionResultTest.java

```diff
@@ -26,7 +26,7 @@ public void setup() {
     }
 
     @Test
-    public void singlePageResults5Datapoints() {
```

Comment:
Make these public

Reply:
That gives Sonar errors that the lead developers don't want

Reply:
Sonar is wrong on this, we need to disable the rule